tests/testthat/test-endpoints.R

```diff
@@ -25,3 +25,22 @@ test_that("Can construct the api", {
   expect_length(logs, 2L)
   expect_identical(logs[[1L]]$logger, "daedalus.api")
 })
+
+test_that("Can get metadata", {
+  endpoint <- daedalus_api_endpoint("GET", "/metadata")
+  res <- endpoint$run()
```

Comment:
Could I suggest a no conditions expectation here:
```suggestion
  expect_no_condition(
    res <- endpoint$run()  
  )
```

Reply:
Good idea, and I've put it on the root call too..

Reply:
Ha, that upsets the linter though: `Avoid implicit assignments in function calls.`
So have excluded those lines from linting!

Reply:
Oh yes, a classic. The other linter complaints can usually be fixed with a `styler::style_pkg()` call